### DISCUSSION TITLE
surfacepro7键盘盖盒盖或者拔掉就会重启ventura

### DISCUSSION THREAD

**Discussion by 1576495283** [badstorm/surface-pro-7-opencore#89] [Q&A]
我是surfacepro7 i5 128G的，黑苹果ventura13.0.1，所有驱动、USB接口我都驱动好了。就是键盘盖盒盖或者拔掉就会重启，不能休眠不说，这个盒盖就重启郁闷死了。求大神解决，万分感谢

**Comment by Rainringgoaway** [comment]
Disable or Wait for Bigsurface new release.

**Reply by 1576495283** [reply]
谢谢

**Comment by zont0x** [comment]
可以用Opencore_Configurator吧Kernel里的voodoo有关项关闭（但保留没有捆绑路径的Bigsurface）这样就正常可以盒盖休眠了（但是Surface自带的触控板只能单指使用了）

**Reply by zont0x** [reply]
可能原因是voodoo可以让macOS识别Surface键盘为内建键盘与触控板 Surface又刚好有个设计缺陷（盒盖后键盘会断联）这就会导致macOS崩溃

**Reply by Rainringgoaway** [reply]
更新bigsurface.kext，新版本已经解决此问题

**Reply by 1576495283** [reply]
> 更新bigsurface.kext，新版本已经解决此问题

好的，谢谢

**Reply by zont0x** [reply]
> 更新bigsurface.kext，新版本已经解决此问题

好的谢谢 亏我研究了这么久😂
